projects/cdk/schematics/ng-update/constants/types.ts

```diff
@@ -0,0 +1,17 @@
+interface TypeToRename {
+    from: string;
+    to: string;
+    moduleSpecifier?: string | string[];
+}
+export const TYPES_TO_RENAME: readonly TypeToRename[] = [
+    {
+        from: 'ButtonOptions',
+        to: 'TuiButtonOptions',
+        moduleSpecifier: ['@taiga-ui/core', '@taiga-ui/core/components'],
```

Comment:
There should be no deep imports in projects. What is the second specifier for? By the way we can probably add a schematic to remove deep imports if projects have any.

Reply:
We need to take not all ButtonOptions, only from concrete package (described in `moduleSpecifier`). I can remove second specifier but who knows how ButtonOptions imported in a random project?)